onnxruntime/core/framework/program_region.h

```diff
@@ -1,13 +1,13 @@
 #pragma once
-#include <vector>
+#include "core/common/inlined_containers.h"
```

Comment:
For return values, you only need forward declarations in the header, so include inlined_containers_fwd.h and include the full header in the .cc file.